eventdisplay/tracks/FairEveGeoTracksEditor.h

```diff
@@ -0,0 +1,60 @@
+/********************************************************************************
+ *    Copyright (C) 2020 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairEveGeoTracksEditor.h
+ *
+ *  Created on: 23 kwi 2020
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef FAIREVEGEOTRACKSEDITOR_H_
+#define FAIREVEGEOTRACKSEDITOR_H_
+
+#include "FairEveCut.h"
+#include "FairEveGeoTracks.h"
+
+#include <GuiTypes.h>
+#include <Rtypes.h>
+#include <RtypesCore.h>
+#include <TGButton.h>
+#include <TGDoubleSlider.h>
+#include <TGFrame.h>
+#include <TGLabel.h>
+#include <TGNumberEntry.h>
+#include <TGedFrame.h>   // for TGedFrame
+#include <memory.h>
```

Comment:
```suggestion
#include <memory>
```

STL headers have no `.h`, see https://en.cppreference.com/w/cpp/header/memory